content/applications/finance/expenses/post_expenses.rst

```diff
@@ -0,0 +1,74 @@
+=============
+Post expenses
+=============
+
+Once an expense report is :ref:`approved <expenses/approve>`, the next step is to post the expense
+report to the proper accounting journal.
+
+.. important::
+   To post expense reports to an accounting journal, the user **must** have the following
+   :doc:`access rights <../../general/users/access_rights>`:
+
+   - Accounting: Accountant or Adviser
+   - Expenses: Manager
```

Comment:
```suggestion
   - **Accounting**: *Accountant* or *Adviser*
   - **Expenses**: *Manager*
```

Reply:
These are not referencing the actual apps, they are instead listing what you see n the Employee form. I do like the italics for the setting, but I will leave the names unbolded.